client/src/main/scala/skuber/Volume.scala

```diff
@@ -17,6 +17,16 @@ object Volume {
       directory: Option[String] = None)
     extends Source
 
+  sealed trait ProjectedSource
+  case class Projected(defaultMode:Option[Int] = None,
+                       sources: List[ProjectedSource]) extends Source
+  case class ProjectedSecret(name: String,
+                             items: Option[List[KeyToPath]] = None) extends ProjectedSource
```

Comment:
The API reference suggests there is an additional (optional) boolean field called `optional` on projected secrets.

Reply:
fixed